presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -25,6 +25,10 @@ Array Functions
 
     Remove duplicate values from the array ``x``.
 
+.. function:: flatten(x) -> array
+
+    Flattens an ``array(array(T))`` to an ``array(T)`` by concatenating its elements.
```

Comment:
Oh, I see. You were thinking of the sub arrays as the things being concatenated. How about "by concatenating the contained arrays"?